bigtable/client/client_options.h

```diff
@@ -53,6 +53,122 @@ class ClientOptions {
     return *this;
   }
 
+  /*
+   * Set Compression Algorithm for channel.
+   *
+   * Please look at
+   * https://grpc.io/grpc/cpp/classgrpc_1_1_channel_arguments.html#aaf332071bbdff5e4a7f3352f4ad564d5
+   * for more details.
+   *
+   */
+  inline void SetCompressionAlgorithm(grpc_compression_algorithm algorithm) {
+    channel_arguments_.SetCompressionAlgorithm(algorithm);
+  }
+
+  /*
+   * Set fallback timeout.
```

Comment:
This comment doesn't add anything to the method, since it's already named "SetGrpcFallbackTimeout".

Please either elaborate further, or remove the comment.

Reply:
Done.